packages/vision/src/index.js

```diff
@@ -157,9 +157,10 @@ Vision.prototype.annotate = function(requests, callback) {
  * For the response in the original format, review the `apiResponse` argument
  * your callback receives.
  *
- * @param {string|string[]|module:storage/file|module:storage/file[]} images -
- *     The source image(s) to run the detection on. It can be either a local
- *     image path, a remote image URL, or a @google-cloud/storage File object.
+ * @param {string|string[]|buffer|buffer[]|module:storage/file|module:storage/file[]} images - The
```

Comment:
I know this looks a bit weird, but dox doesn't parse it quite right if all the types, the name and at least part of the description are on a single line.